### PR TITLE
docs: kotlin core api reference(zh-cn)

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

* docs: kotlin core api reference(zh-cn)
* update kotlin core sdk reference (remove redundant docs)

## Linear Issue Reference

* LOG-2278

## Testing

image

### CODE REVIEW

File: packages/docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/sdk/Kotlin/kotlin/io.logto.sdk.core.type/-refresh-token-token-response/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# RefreshTokenTokenResponse
+
+
+data class RefreshTokenTokenResponse(val accessToken: [String](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html), val refreshToken: [String](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html), val idToken: [String](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html)?, val scope: [String](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html), val expiresIn: [Long](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-long/index.html))
+
+## Constructors
+
+| | |
```
Comment (IceHe) [inline packages/docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/sdk/Kotlin/kotlin/io.logto.sdk.core.type/-refresh-token-token-response/index.md:8]: Empty Header?

File: packages/docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/sdk/Kotlin/kotlin/io.logto.sdk.core.type/-oidc-config-response/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# OidcConfigResponse
+
+
+data class OidcConfigResponse(val authorizationEndpoint: [String](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html), val tokenEndpoint: [String](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html), val endSessionEndpoint: [String](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html), val userinfoEndpoint: [String](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html), val jwksUri: [String](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html), val issuer: [String](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html), val revocationEndpoint: [String](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html))
+
+## Constructors
+
+| | |
```
Comment (IceHe) [inline packages/docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/sdk/Kotlin/kotlin/io.logto.sdk.core.type/-oidc-config-response/index.md:8]: Empty header?

File: packages/docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/sdk/Kotlin/kotlin/io.logto.sdk.core.type/-id-token-claims/index.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# IdTokenClaims
+
+
+data class IdTokenClaims(val iss: [String](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html), val sub: [String](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html), val aud: [String](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html), val exp: [Long](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-long/index.html), val iat: [Long](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-long/index.html), val atHash: [String](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html)?)
+
+## Constructors
+
+| | |
```
Comment (IceHe) [inline packages/docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/sdk/Kotlin/kotlin/io.logto.sdk.core.type/-id-token-claims/index.md:8]: empty header?

File: packages/docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/sdk/Kotlin/kotlin/io.logto.sdk.core.type/-code-token-response/index.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# CodeTokenResponse
+
+
+data class CodeTokenResponse(val accessToken: [String](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html), val refreshToken: [String](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html), val idToken: [String](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html), val scope: [String](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html), val expiresIn: [Long](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-long/index.html))
+
+## Constructors
+
+| | |
```
Comment (IceHe) [inline packages/docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/sdk/Kotlin/kotlin/io.logto.sdk.core.type/-code-token-response/index.md:8]: empty header?

**Review by IceHe — APPROVED** [review]
lgtm

**Comment by demonzoo** [comment]
Is this auto generated by a tool? If yes, then I won't bother going into the details (TL;DR)

**Comment by xiaoyijun** [comment]
> Is this auto generated by a tool? If yes, then I won't bother going into the details (TL;DR)

Yes. @demonzoo 
Reference: https://github.com/Kotlin/dokka

**Review by demonzoo — APPROVED** [review]

**Review by gao-sun — CHANGES_REQUESTED** [review]
should be 2 PRs for reorg and doc adding?

**Review by gao-sun — APPROVED** [review]
lets separate next time

**Pull request merged**